src/core/screen.js

```diff
@@ -81,6 +81,9 @@ export class Screen extends Phaser.Scene {
         data.config && this.events.once("create", nextPage(this));
 
         if (this.scene.key !== "loader" && this.scene.key !== "boot") {
+            if (!this.scene.key.includes("shop")) {
+                this.setStatsScreen(this.scene.key);
+            }
```

Comment:
not a big fan of basing these things off scene key but I guess it's the easiest way.
you could prob make this into one big if

Reply:
Yeah I think scene key is maybe the best way handle multiple scenes.

Worth putting anything we might want to refactor at a later date into an issue on github.

Reply:
!["loader", "boot"].includes(this.scene.key)